uncontrast_studies/management/commands/load_uncon_data.py

```diff
@@ -33,19 +33,28 @@
 
 logger = logging.getLogger(__name__)
 
-FILE_PATH = "uncontrast_studies/data/dataset_16062024.xlsx"
-current_date = date.today()
-ERROR_LOG_PATH = f"uncontrast_studies/data/logs/uncontrast_errors_log_{current_date}.xlsx"
-SUCCESS_LOG_PATH = f"uncontrast_studies/data/logs/uncontrast_success_log_{current_date}.xlsx"
-
 
 class Command(BaseCommand):
-    help = "Load uncon existing data"
+    help = "Load uncontrast existing data"
+
+    def add_arguments(self, parser):
+        parser.add_argument("-f", "--filename", type=str, help="Filename to load")
```

Comment:
it's not optional?

Reply:
it is, I thought this was the syntax for optional arguments